app/police-custody-form/controllers.js

```diff
@@ -4,7 +4,124 @@ exports.addEvents = async function (req, res) {
   const user = req.user
   const move = await req.services.move.getById(moveId)
 
+  if (lockoutEvents.events === undefined) {
```

Comment:
What scenario is this catching?

Reply:
When the user just submits an empty form. With no checkboxes ticked or details entered